### DISCUSSION TITLE
Logging into a specific channel

### DISCUSSION THREAD

**Discussion by steverob1066** [tegal1337/YOMEN#31] [Q&A] [answered]
I'm sorry if this is a stupid question or I missed it, but how do you log in as a specific Youtube channel? When I set the Google login parameters there's no option to select anything other than the default channel.

**Comment by fdciabdul** [comment, marked as answer]
not yet added in this , btw you can switch default channel to second channel 

https://support.google.com/youtube/answer/6019090?hl=en
